### ISSUE TITLE
Not working in Unity 2017

### ISSUE THREAD

**Issue by bnussey** [ssannandeji/Zenject-2019#297]
When loading up Zenject project with Unity 2017 I get the following:

```
Assets/Plugins/Zenject/Source/Editor/Testing/ZenjectIntegrationTestFixture.cs(81,51): error CS1061: Type `NUnit.Framework.TestContext.ResultAdapter' does not contain a definition for `Status' and no extension method `Status' of type `NUnit.Framework.TestContext.ResultAdapter' could be found. Are you missing an assembly reference?
Assets/Plugins/Zenject/Source/Editor/Testing/ZenjectIntegrationTestFixture.cs(85,46): error CS0246: The type or namespace name `ExpectedExceptionAttribute' could not be found. Are you missing an assembly reference?
```

**Comment by svermeulen** [comment]
This should be fixed in the new release (5.2.1), which is still under review with asset store.  In the meantime, you can grab the latest from the [releases section](https://github.com/modesttree/Zenject/releases)

**Comment by bnussey** [comment]
Wow fast release time @svermeulen .. really appreciate the support and loving Zenject. Thank you.
